codec-classes-quic/src/main/java/io/netty/incubator/codec/quic/QuicTokenHandler.java

```diff
@@ -51,4 +51,9 @@ public interface QuicTokenHandler {
      * @return the maximal supported token length.
      */
     int maxTokenLength();
+
+    /** @return Maximum possible length of the {@code dcid} buffer passed to {@link #writeToken} method */
+    static int maxConnectionIdLength() {
```

Comment:
I think we should better not use `Quiche.*` within here as it might result into problems when the native lib is not loaded.

Reply:
But same kind of problems will arise when `InsecureQuicTokenHandler` is instantiated. What are the other alternatives?

This definitely should be exposed one way or another, otherwise it is impossible to write any third-party implementation for the `QuicTokenHandler` cleanly. You need that to calculate the maximum token size.